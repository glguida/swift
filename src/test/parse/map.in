random Ball x ~ Categorical({b[0]->0.5, b[1]->0.5});