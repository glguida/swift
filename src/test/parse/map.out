(BlogProgram:
  (args:
    (FuncDecl:
      :kind random
      :type Ball
      :func x
      expr:
        (DistrExpr:
          :distr Categorical
          (args:
            (MapExpr:
              :from#0
                (OpExpr:
                  :op 16
                  :left
                    (VarRef: b )
                  :right
                    (IntLiteral: 0 )
                )
              :to#0
                (DoubleLiteral: 0.500000 )
              :from#1
                (OpExpr:
                  :op 16
                  :left
                    (VarRef: b )
                  :right
                    (IntLiteral: 1 )
                )
              :to#1
                (DoubleLiteral: 0.500000 )
            )
          )
        )
    )
  )
)
